floodviz/templates/index.html

```diff
@@ -16,6 +16,16 @@
 
 	</script>
 	<script type="text/javascript">
+
+		// Google Analytics ID
```

Comment:
There is an easier way to do this. Use a template if block around the code that sets up the google analytics. See http://jinja.pocoo.org/docs/2.9/templates/#tests. This will set up google analytics except without any code which is not good. You will also want to use an if block in ga_send_event to do nothing if GA_ID is null.